packages/cardano-services-client/test/HandleProvider/handleProvider.test.test.ts

```diff
@@ -0,0 +1,29 @@
+import { handleHttpProvider } from '../../src';
```

Comment:
I think the file extension has one too many `.test`

Reply:
3e3a6b1a1 should solve this